src/python/WMCore/MicroService/MSOutput/MSOutput.py

```diff
@@ -1023,3 +1027,20 @@ def alertDatatierNotFound(self, datatierName, containerName, isRelVal):
         self.logger.critical(alertDescription)
         if self.msConfig["sendNotification"] and not isRelVal:
             self.sendAlert(alertName, alertSeverity, alertSummary, alertDescription, self.alertServiceName)
+
+    def alertGenericError(self, caller, workflowname, msg, ex_str, document_str):
+        """
+        Send an alert in the case of a generic error with ms-output 
```

Comment:
Please improve this docstring. You might want to refer to this section in the contribution guidelines:
https://github.com/dmwm/WMCore/blob/master/CONTRIBUTING.rst#project-docstrings-best-practices

Reply:
oook :)